src/main/java/com/toofifty/easyblastfurnace/methods/Method.java

```diff
@@ -23,8 +23,8 @@
 
     protected final MethodStep withdrawIceOrSmithsGloves = new ItemStep(ItemID.ICE_GLOVES, ItemID.SMITHS_GLOVES_I, "Withdraw ice gloves or smiths gloves (i)");
     protected final MethodStep equipIceOrSmithsGloves = new ItemStep(ItemID.ICE_GLOVES, ItemID.SMITHS_GLOVES_I, "Equip ice gloves or smiths gloves (i)");
-    protected final MethodStep withdrawGoldsmithGauntlets = new ItemStep(ItemID.GOLDSMITH_GAUNTLETS, "Withdraw goldsmith gauntlets");
-    protected final MethodStep equipGoldsmithGauntlets = new ItemStep(ItemID.GOLDSMITH_GAUNTLETS, "Equip goldsmith gauntlets");
+    protected final MethodStep withdrawGoldsmithGauntlets = new ItemStep(ItemID.GOLDSMITH_GAUNTLETS, "Withdraw goldsmith gauntlets or Smithing cape");
+    protected final MethodStep equipGoldsmithGauntlets = new ItemStep(ItemID.GOLDSMITH_GAUNTLETS, "Equip goldsmith gauntlets or Smithing cape");
```

Comment:
We should be able to check their bank for the smithing cape - I think it'd make sense to have withdraw/equip cape as separate steps.

Could also then make use of the alternate ID-
```java
new ItemStep(ItemID.SMITHING_CAPE, ItemID.SMITHING_CAPET, "Withdraw Smithing cape");
```

(don't ask me why I didn't make it an array)